examples/ads.amp.html

```diff
@@ -1194,6 +1194,12 @@ <h2>Rubicon Project Smart Tag</h2>
       json='{"visitor":{"age":"18-24","gender":"male"},"inventory":{"section":"amp"}}'>
   </amp-ad>
 
+  <h2>Sekindo</h2>
```

Comment:
Please add your example to this [form](https://github.com/ampproject/amphtml/blob/master/examples/ads.amp.html#L56) as well.
Also please sort them in alphabetic order. This goes after `rubicon`